oci/pull.bzl

```diff
@@ -403,16 +358,8 @@ oci_pull_rule = repository_rule(
         "platform": attr.string(doc = "platform in `os/arch` format, for multi-arch images"),
         "target_name": attr.string(doc = "Name given for the image target, e.g. 'image'", mandatory = True),
         "toolchain_name": attr.string(default = "oci", doc = "Value of name attribute to the oci_register_toolchains call in the workspace."),
+        "config": attr.label(doc = "Label to a config.json file", default = "@oci_config//:config.json"),
```

Comment:
docstring could say "internal use only" or "it should be generated by XXX" so the docs/*.md can help someone understand the layering of macros/rules